src/components/composer/composer/composer.ts

```diff
@@ -447,6 +447,32 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
     this.processTokenAtCursor();
   }
 
+  onDblClick() {
+    if (this.env.model.getters.isReadonly()) {
+      return;
+    }
+    const composerContent = this.env.model.getters.getCurrentContent();
+    const isValidFormula =
+      composerContent.startsWith("=") && this.env.model.getters.getCurrentTokens().length > 0;
+
+    if (isValidFormula) {
+      const tokens = this.getColoredTokens();
```

Comment:
can't you use `getters.getCurrentTokens()` instead of `getColoredTokens` here ?
It looks weird since this it has nothing to do with colors. (even though `getColoredTokens` is wrongly named in the first place since it doesn't return tokens)